src/pages/home/report/ReportActionCompose/ReportActionCompose.js

```diff
@@ -288,30 +285,11 @@ function ReportActionCompose({
         setIsFocused(true);
     }, []);
 
-    /**
-     * Used to show Popover menu on Workspace chat at first sign-in
-     * @returns {Boolean}
-     */
-    const showPopoverMenu = useCallback(() => {
-        setMenuVisibility(true);
-        return true;
-    }, []);
-
     useEffect(() => {
-        // Shows Popover Menu on Workspace Chat at first sign-in
-        if (!disabled) {
-            Welcome.show({
-                routes: lodashGet(navigation.getState(), 'routes', []),
-                showPopoverMenu,
-            });
+        if (!EmojiPickerActions.isActive(report.reportID)) {
```

Comment:
This block i.e. useEffect without dependencies was working similar to componentWillUnmount, I think.
We should keep it in `return ()` to maintain the previous behavior

Reply:
I have tested the emoji picker and the behaviour is still the same